### ISSUE TITLE
Trouble pip3 installing the requirements - flask_ask error

### ISSUE THREAD

**Issue by llewmihs** [stevenleeg/geemusic#117]
I'm running into a snag along the tutorial steps.

When I run:
`pip3 install -r requirements.txt`

Some of the package install, but I get this error:
`Command /usr/bin/python3 -c "import setuptools, tokenize;__file__='/tmp/pip-build-xu1qf3qx/cryptography/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-ljnm0g3w-record/install-record.txt --single-version-externally-managed --compile failed with error code 1 in /tmp/pip-build-xu1qf3qx/cryptography
Storing debug log for failure in /root/.pip/pip.log`

So then when I load foreman I get this:
`22:06:28 web.1  | started with pid 11151
22:06:31 web.1  | Traceback (most recent call last):
22:06:31 web.1  |   File "server.py", line 1, in <module>
22:06:31 web.1  |     from geemusic import app
22:06:31 web.1  |   File "/home/pi/geemusic/geemusic/__init__.py", line 3, in <module>
22:06:31 web.1  |     from flask_ask import Ask
22:06:31 web.1  |   File "/usr/local/lib/python3.4/dist-packages/flask_ask/__init__.py", line 9, in <module>
`
I've tried running the pip3 install in a virtual env, as sudo and as su. But all with the same issue.

I'm using a PiZeroW with latest Raspbian install.

Any help, greatly appreciated.

Llewmihs

**Comment by stevenleeg** [comment]
This looks like an issue with your local environment rather than something with geemusic itself. My best guess would be that you don't have the `build-essential` package installed (eg `gcc`)?

Either way, I'll have to close this issue as it's off-topic from the project.

**Comment by llewmihs** [comment]
If it helps I solved the problem:
When using pip3 with the `requirements.txt` file the install fails on the latest version of Raspbian.
Solution:
I removed the version numbers from the pip3 packages in the `requirements.txt`, installing only latest versions. Problem solved, all packages install successfully.
This seems to be a Raspbian-specific issue, as the packages install successfully on Ubunutu Mate.
